papaparse.js

```diff
@@ -582,6 +582,16 @@
 
 			xhr.open('GET', this._input, !IS_WORKER);
 
+			if (this._config.downloadRequestHeaders)
```

Comment:
Can we add a comment to indicate that the state must be OPEN to set the request?